todo_app/app.py

```diff
@@ -1,11 +1,19 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
```

Comment:
Very minor, but worth noting that you can combine these two lines if you prefer, with:
```suggestion
from todo_app.data.session_items import get_items, add_item
```